windows/deploy_windows.ps1

```diff
@@ -159,7 +159,7 @@ Function Install-Dependencies
     }
     Initialize-Module-Here -m "VSSetup"
     Install-Dependency -Uri $NsisUrl `
-        -Name $NsisName -Destination "NSIS"
+        -Name $NsisName -Destination "..\libs\NSIS\NSIS-source"
```

Comment:
Why should this be different from line 328?

Reply:
I tested that and it seemed to just append the value of $RootPath to the working directory